### DISCUSSION TITLE
namespaces in vcsim

### DISCUSSION THREAD

**Discussion by olzemal** [vmware/govmomi#3361] [Community Help] [answered]
Is it possible to simulate vSphere namespaces with vcsim? 

I tried running the container but i get a 404 error.
```console
$ docker run -d -p 8989:8989 vmware/vcsim:latest
$ curl -k https://localhost:8989/api/vcenter/namespaces/instances
404 page not found
```
According to the [docs](https://developer.vmware.com/apis/vsphere-automation/latest/vcenter/api/vcenter/namespaces/instances/get/) i expected an empty array to return with status 200.

**Comment by dougm** [comment]
Hi @olzemal , you should get the expected result with #3397 merged. Will be included in the next release

**Comment by dougm** [comment, marked as answer]
Released: https://github.com/vmware/govmomi/releases/tag/v0.37.0
